sbin/start_l1.sh

```diff
@@ -15,49 +15,84 @@ fi
 echo "Using dotenv: $ENV"
 . $ENV
 
-echo "Force-removing l1_geth container if it exists..."
-docker rm --force l1_geth
-
 echo "Removing localhost deployment artifacts..."
 rm -rf $CONTRACTS_DIR/deployments/localhost
 
-# Start L1 network.
-echo "Starting L1..."
+CWD=`pwd`
 L1_WS_PORT=`echo $L1_ENDPOINT | awk -F':' '{print $3}'`
 echo "Parsed port: $L1_WS_PORT from $L1_ENDPOINT"
-docker run -d \
-  --name l1_geth \
-  -p $L1_WS_PORT:$L1_WS_PORT \
-  ethereum/client-go \
-  --dev \
-  --dev.period 1 \
-  --verbosity 3 \
-  --http \
-  --http.api eth,web3,net \
-  --http.addr 0.0.0.0 \
-  --ws \
-  --ws.api eth,net,web3 \
-  --ws.addr 0.0.0.0 \
-  --ws.port $L1_WS_PORT
-sleep 3
-
-echo "Funding addresses..."
-docker exec l1_geth geth attach --exec \
-  "eth.sendTransaction({ from: eth.coinbase, to: '"$SEQUENCER_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
-  $L1_ENDPOINT
-docker exec l1_geth geth attach --exec \
-  "eth.sendTransaction({ from: eth.coinbase, to: '"$VALIDATOR_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
-  $L1_ENDPOINT
-docker exec l1_geth geth attach --exec \
-  "eth.sendTransaction({ from: eth.coinbase, to: '"$DEPLOYER_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
-  $L1_ENDPOINT
 
-CWD=`pwd`
+###### PID handling ######
+trap ctrl_c INT
+
+# Active PIDs
+PIDS=()
+
+function cleanup() {
+    echo "Cleaning up..."
+    for pid in "${PIDS[@]}"; do
+	echo "Killing $pid"
+	kill $pid
+    done
+}
+
+function ctrl_c() {
+    cleanup
+}
+##########################
+
+# Start L1 network.
+echo "Starting L1..."
+if [ $L1_STACK = "geth" ]; then
+    echo "Force-removing l1_geth container if it exists..."
+    docker rm --force l1_geth
+    docker run -d \
+      --name l1_geth \
+      -p $L1_WS_PORT:$L1_WS_PORT \
+      ethereum/client-go \
+      --dev \
+      --dev.period 1 \
+      --verbosity 3 \
+      --http \
+      --http.api eth,web3,net \
+      --http.addr 0.0.0.0 \
+      --ws \
+      --ws.api eth,net,web3 \
+      --ws.addr 0.0.0.0 \
+      --ws.port $L1_WS_PORT
+    sleep 3
+
+    echo "Funding addresses..."
```

Comment:
i think we could wait for these transactions in parallel to speed up initializing the L1, but not necessary for this PR. could look something like this:
```
docker exec l1_geth geth attach --exec \
  "eth.sendTransaction({ from: eth.coinbase, to: '"$SEQUENCER_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
  $L1_ENDPOINT & 
PID1=$!

docker exec l1_geth geth attach --exec \
  "eth.sendTransaction({ from: eth.coinbase, to: '"$VALIDATOR_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
  $L1_ENDPOINT & 
PID2=$!

docker exec l1_geth geth attach --exec \
  "eth.sendTransaction({ from: eth.coinbase, to: '"$DEPLOYER_ADDRESS"', value: web3.toWei(10000, 'ether') })" \
  $L1_ENDPOINT & 
PID3=$!

# Wait only for the background jobs started by this script
wait $PID1 $PID2 $PID3
```